### PR TITLE
Static ips

### LINKED ISSUE #1: Use static IPs for boxes
By using static IPs at least we prevent the need to rebind services when doing updates to the deployment.

### PR DESCRIPTION
**Pull request by jmcarp**
[Resolves #1]

cc @afeld @dlapiduz

### CODE REVIEW

**Comment by dlapiduz** [comment]
@jmcarp did you upload the secrets?

**Comment by jmcarp** [comment]
@dlapiduz yep, I updated the secrets in the staging bucket, re-ran the pipeline, and verified that we can still update the broker and create service instances.

File: bosh/secrets.example.yml [hunk 1]
```diff
@@ -34,12 +33,40 @@ properties:
...
     - sec_group
     subnet: subnet
+  static:
+  - <changme>
+  dns:
+  - <changeme>
+  gateway: <changeme>
+  reserved:
+  - <changeme>
+  range: <changeme>
```
Comment (afeld) [inline bosh/secrets.example.yml:50]: Are all of the existing IPs in a certain range? Do we need to check for or exclude other instances from being created in that range? `/me doesn't actually understand networking`

File: bosh/secrets.example.yml [hunk 2]
```diff
@@ -34,12 +33,40 @@ properties:
     component_name: cf-swarm-containers-broker
     docker_url: secret
 
-
 networks:
 - name: default
-  type: dynamic
+  type: manual
```
Comment (afeld) [inline bosh/secrets.example.yml:38]: This is preferable to using a `dynamic` `default` network, then making a separate one with `type:vip`?

Comment (jmcarp) [inline bosh/secrets.example.yml:38]: I tried using two `network`s and wound up hitting timeouts on deploy, which is why I went this direction. But there's a non-trivial chance that I just set it up wrong, so we can take another look if you're interested.

File: bosh/secrets.example.yml (hunk 1 above)
Comment (jmcarp) [inline bosh/secrets.example.yml:50]: IIRC, bosh will die if we ask for overlapping ranges. To get this working on staging, I picked a range that included the current IPs of the broker and docker machines.

**Pull request merged**